lib/presentation/views/about_us/about_us_info.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class AboutUsInfoView extends StatelessWidget {
+  final String message;
```

Comment:
Can be removed

Reply:
not sure what line you're refering to remove sorry, as in the whole file?

Reply:
done